source/form.html

```diff
@@ -10,11 +10,11 @@
       <a class="header__logo logo" href="index.html">
         <img src="img/logo.svg" width="74" height="23" alt="Логотип Пинк.">
       </a>
-      <nav class="header-nav">
-        <button class="header-nav__toggle" type="button">
+      <nav class="header__menu">
+        <button class="header__menu__toggle" type="button">
```

Comment:
А что произошло с файлами кроме главной страницы? Тут какая-то другая разметка по сравнению с главной, что не есть хорошо.
А еще тут по БЭМ идет элемент элемента. Так быть не должно:
```suggestion
        <button class="header__toggle" type="button">
```

И для `header__menu__list` тоже надо править